src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const orderedStyles = sourceString.split(';');
+
+  const cssObject = {};
+
+  orderedStyles.forEach((style) => {
+    const [key, value] = style.split(':');
+
+    if (key && value) {
+      cssObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a loop with `forEach`, consider using array methods like `map` and `filter` to transform the array of styles into the desired object. This approach is more declarative and aligns with functional programming principles.